src/config/index.ts

```diff
@@ -10,6 +10,12 @@ export default {
     brazeApiKey:
       process.env.BRAZE_API_KEY || 'BrazeContentProxy/Dev/BRAZE_API_KEY',
   },
+  // The URL to query data from.
+  clientApi: {
+    uri: `https://client-api.getpocket.${
+      process.env.NODE_ENV === 'production' ? 'com' : 'dev'
```

Comment:
nit / later cleanup: use `environment` above so we don't have to check `process.env.NODE_ENV` in two places/two ways.